environment/scripts/prepare.sh

```diff
@@ -20,9 +20,6 @@ then
 echo "near-core home not specified..."
 git clone "https://github.com/nearprotocol/nearcore" $CORE_SRC
 eval CURR_DIR=$(pwd)
-cd $CORE_SRC
-# Freeze nearcore version to avoid RPC changes breaking tests.
-git checkout df431768f95bc4117dc7c4d954e38f1b96fd3b9f
```

Comment:
we might consider bin search which commit cause near proof invalid in near prover

Reply:
Weird. It passes now. Maybe it was a fluke.